facades/PC/src/main/java/org/terasology/engine/GLFWSplashScreen.java

```diff
@@ -55,6 +54,47 @@ public void close() {
 
     @Override
     public void run() {
+        Renderer renderer = Mono.just(0)
+                .subscribeOn(GameScheduler.graphics())
+                .doOnNext(n -> initWindow())
+                .map((n) -> new Renderer())
+                .doOnNext(Renderer::init)
```

Comment:
Starting this with `just(0)` instead of `initWindow` feels weird. Is that so you have something to `subscribeOn`? Is this the sort of thing that `publishOn` is for?

Reply:
I detected mono.fromRunable method too late.

All methods used there should runs at graphics.